src/components/Channel/Channel.js

```diff
@@ -136,6 +138,7 @@ const ChannelInner = ({
   const originalTitle = useRef('');
   const lastRead = useRef(new Date());
   const chatContext = useContext(ChatContext);
+  const [online, setOnline] = useState(true);
```

Comment:
we can use ref instead of state here to avoid causing re-render since it's not necessary